src/main/java/com/scaleset/cfbuilder/ec2/metadata/CFNService.java

```diff
@@ -0,0 +1,39 @@
+package com.scaleset.cfbuilder.ec2.metadata;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.*;
+
+public class CFNService {
+
+    @JsonIgnore
+    public String id;
+
+    private Map<String, SimpleService> services;
+
+    public CFNService() {
+        //Hardcoded because we only support sysvinit atm
+        this.id = "sysvinit";
+        this.services = new HashMap<>();
+    }
+
+    public CFNService(String id) {
```

Comment:
Is this needed?
If it's a placeholder for later versions, then keep it, otherwise I think it can be removed.

Reply:
If we ever support windows the id can't be `sysvinit', so for later versions this might be needed